### PR TITLE
Create README-nl.md

### PR DESCRIPTION
**Pull request by SilvinvanHaestregt**
Added the Dutch translation for the README file.

### CODE REVIEW

**Comment by ORelio** [comment]
Thanks for your contribution!
Is there anyone here would could help proofreading this translation?
(Otherwise I'll just merge later, don't worry)

**Comment by JesperKuipers** [comment]
ill read it

File: README-nl.md [hunk 1]
```diff
@@ -10,64 +10,64 @@ Minecraft Console Client (MCC) is een simpele app waarmee je Minecraft servers k
...
 
 Bekijk de [sample configuration files](MinecraftClient/config/) die de how-to-use bevat [README](https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config#minecraft-console-client-user-manual).
 
-## Getting Help 🙋
+## Hulp Krijgen 🙋
 
 Bekijk de [README](https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config#minecraft-console-client-user-manual) en de bestaande [Discussions](https://github.com/MCCTeam/Minecraft-Console-Client/discussions): Misschien is je vraag daar al beantwoord. Als dat niet zo is open dan een [New Discussion](https://github.com/MCCTeam/Minecraft-Console-Client/discussions/new) en stel je vraag. Als je een bug vind, raporteer dat in de [Issues](https://github.com/MCCTeam/Minecraft-Console-Client/issues) sectie.
 
-## Helping Us ❤️
+## Ons helpen ❤️
```
Comment (JesperKuipers) [inline README-nl.md:21]: "Steentje bijdragen" sounds better, this isnt wrong tho

Comment (SilvinvanHaestregt) [reply README-nl.md:21]: Sounds allot better I will change it.

File: README-nl.md [hunk 2]
```diff
@@ -10,64 +10,64 @@ Minecraft Console Client (MCC) is een simpele app waarmee je Minecraft servers k
 Krijg het binary bestand van onze laatste [development build](https://github.com/MCCTeam/Minecraft-Console-Client/releases/latest).
```
Comment (JesperKuipers) [inline README-nl.md:10]: "Verkrijg [hier](https://github.com/MCCTeam/Minecraft-Console-Client/releases/latest) de laatste ontwikkelde binaire versie.

Comment (SilvinvanHaestregt) [reply README-nl.md]: I just changed it and i think it is completely okay now.

**Comment by JesperKuipers** [comment]
@SilvinvanHaestregt how does this look? https://github.com/SilvinvanHaestregt/Minecraft-Console-Client/pull/1

**Comment by JesperKuipers** [comment]
@ORelio I approve this PR

**Comment by milutinke** [comment]
Awesome.
By the way, we have a documentation [website](https://mccteam.github.io) now.
If you're willing and you have time, you could contribute to it by translating it.
It is written using [Vue Press v2](https://v2.vuepress.vuejs.org/).
[Here](https://v2.vuepress.vuejs.org/guide/i18n.html#site-i18n-config) you can find out how to add another language, it's simple.

**Comment by SilvinvanHaestregt** [comment]
> Awesome. By the way, we have a documentation [website](https://mccteam.github.io) now. If you're willing and you have time, you could contribute to it by translating it. It is written using [Vue Press v2](https://v2.vuepress.vuejs.org/). [Here](https://v2.vuepress.vuejs.org/guide/i18n.html#site-i18n-config) you can find out how to add another language, it's simple.
> 
> The documentation repository: https://github.com/MCCTeam/MCCTeam.github.io

I would love to do that however I was reading the documentation and this is not in your config file

`module.exports = {
  locales: {
    // The key is the path for the locale to be nested under.
    // As a special case, the default locale can use '/' as its path.
    '/': {
      lang: 'en-US',
      title: 'VuePress',
      description: 'Vue-powered Static Site Generator',
    },
    '/zh/': {
      lang: 'zh-CN',
      title: 'VuePress',
      description: 'Vue 驱动的静态网站生成器',
    },
  },
}`

**Comment by milutinke** [comment]
> > Awesome. By the way, we have a documentation [website](https://mccteam.github.io) now. If you're willing and you have time, you could contribute to it by translating it. It is written using [Vue Press v2](https://v2.vuepress.vuejs.org/). [Here](https://v2.vuepress.vuejs.org/guide/i18n.html#site-i18n-config) you can find out how to add another language, it's simple.
> > The documentation repository: https://github.com/MCCTeam/MCCTeam.github.io
> 
> I would love to do that however I was reading the documentation and this is not in your config file
> 
> `module.exports = { locales: { // The key is the path for the locale to be nested under. // As a special case, the default locale can use '/' as its path. '/': { lang: 'en-US', title: 'VuePress', description: 'Vue-powered Static Site Generator', }, '/zh/': { lang: 'zh-CN', title: 'VuePress', description: 'Vue 驱动的静态网站生成器', }, }, }`
> 
> And I am maybe wrong but the file structure is not correct in the docs map for multiple languages.

Yeah, because I haven't had any other languages.
You can add it to the file.

**Comment by ORelio** [comment]
@JesperKuipers Thanks for helping in proofreading 👍
@SilvinvanHaestregt Thanks again for contributing!

**Pull request merged**